app/questionnaire/questionnaire_schema.py

```diff
@@ -454,6 +454,14 @@ def _populate_answer_and_list_dependencies(self) -> None:
                 self._update_dependencies_for_summary(block)
                 continue
 
+            if block["type"] == "ListCollectorContent" and block.get(
+                "repeating_blocks"
+            ):
+                # normal list collectors don't need this because the add/remove handlers manage revisiting repeating blocks
```

Comment:
Really minor, but would say something like `ListCollector block types`, rather than `normal list collectors` to make this clearer and more explicit

Reply:
Changed to "Editable" list collectors as I wonder if list collector block type might imply that its any of the list collector blocks and confuse people? but I think editable works and is more explicit than this. Seem ok?